Assets/MixedRealityToolkit.Tools/ExtensionServiceCreator/ExtensionServiceCreator.cs

```diff
@@ -85,6 +81,38 @@ private struct PersistentState
 
         #endregion
 
+        #region paths
+
+        private string ExtensionsFolder
+        {
+            get
```

Comment:
nit: inconsistent syntax between this (`get { }`) and the ones following (`get =>`).

Could we turn them all into `private string Foo => MixedRealityToolkitFiles.Bar()`? :)

Reply:
i'll update the syntax :)

Reply:
fixed